test/integration/scheduler_perf/test-performance.sh

```diff
@@ -43,10 +45,10 @@ if ${RUN_BENCHMARK:-false}; then
   go test -c -o "perf.test"
 
   kube::log::status "performance test (benchmark) start"
-  "./perf.test" -test.bench=. -test.run=xxxx -test.cpuprofile=prof.out -test.short=false
+  "./perf.test" $glog_flags -test.bench=. -test.run=xxxx -test.cpuprofile=prof.out -test.short=false
   kube::log::status "...benchmark tests finished"
 fi
 # Running density tests. It might take a long time.
 kube::log::status "performance test (density) start"
-go test -test.run=. -test.timeout=60m -test.short=false
+go test $glog_flags -test.run=. -test.timeout=60m -test.short=false
```

Comment:
these are not being passed correctly.

Reply:
please see https://github.com/kubernetes/kubernetes/pull/79242 for how to correctly pass arbitrary args to the test code

Reply:
https://github.com/kubernetes/kubernetes/pull/79283 is a better example with more details